docs/unittesting.md

```diff
@@ -50,15 +116,15 @@ Remix facilitates users with various types of customizations to test a contract
 
 **1. Custom Compiler Context**
 
-`Solidity Unit Testing` refers `Solidity Compiler` plugin for compiler configurations. One can provide customized inputs for `Compiler`, `EVM Version` & `Enable Optimization` and these will be the configuration settings used for contract compilation before running unit tests.
+`Solidity Unit Testing` refers `Solidity Compiler` plugin for compiler configurations. One can provide customized inputs for `Compiler`, `EVM Version`, `Enable Optimization` & `runs`. This configuration will be used for contract compilation before running unit tests.
 
 ![](images/a-unit-testing-custom-compiler-config.png)
 
 **2. Custom Transaction Context**
 
-For a contract method interaction, prime parameters of transaction are `from` address, `value` & `gas`. Usually, we need to test a method's behaviour under different values of these parameters.
+For interacting with a contract's method, the prime parameters of a transaction are `from` address, `value` & `gas`. Typically, a method's behaviour is tested with different values of these parameters.
 
-Remix provides the functionality of custom `msg.sender` & `msg.value` of transaction using method devdoc like:
+Remix provides the functionality of custom `msg.sender` & `msg.value` of transaction using natspec, like:
```

Comment:
One can input  custom values for `msg.sender` & `msg.value` of transaction using NatSpec comments, like:

Reply:
updated